deps/rabbitmq_peer_discovery_aws/src/rabbit_peer_discovery_aws.erl

```diff
@@ -370,9 +372,78 @@ get_value(_, []) ->
 get_value(Key, Props) when is_integer(Key) ->
     {"item", Props2} = lists:nth(Key, Props),
     Props2;
+get_value("networkInterfaceSet", Props) ->
```

Comment:
Perhaps you could use a more generic sort function that behaves differently depending on key, instead of mapping the keys up here? And leave "get_value" untouched.